lego/apps/events/models.py

```diff
@@ -132,6 +132,17 @@ def user_should_see_regs(self, user: User) -> bool:
             or user.is_abakom_member
             or (self.created_by is not None and self.created_by.id == user.id)
         )
+    
+    def user_should_see_allergies(self, user: User) -> bool:
+        memberships = Membership.objects.filter(user=user)
+        in_responsible_group = self.responsible_group in [mem.abakus_group for mem in memberships]
```

Comment:
Is it not better to just check if the user is part of the responsible group than to go through all memberships and see if the group is there?

Reply:
I guess? I just assumed that user's have a "friendship paradox" relation to groups. As in: The average group has more members than the average member has groups